owncloudApp/src/main/java/com/owncloud/android/presentation/ui/authentication/LoginActivity.kt

```diff
@@ -634,16 +637,17 @@ class LoginActivity : AppCompatActivity(), SslUntrustedCertDialog.OnSslUntrusted
     }
 
     private fun initBrandableOptionsUI() {
-        if (!contextProvider.getBoolean(R.bool.show_server_url_input)) {
-            binding.hostUrlFrame.isVisible = false
-            binding.centeredRefreshButton.run {
-                isVisible = true
-                setOnClickListener { checkOcServer() }
-            }
+        val preferencesProvider = SharedPreferencesProviderImpl(this)
+        val showInput = if (BuildConfig.FLAVOR == "mdm") preferencesProvider.getBoolean(CONFIGURATION_SERVER_URL_INPUT_VISIBILITY, true) else contextProvider.getBoolean(R.bool.show_server_url_input)
```

Comment:
I've seen you use the string `"mdm"` several times, maybe it could be a variable you can call. What if the flavor name changes?

Reply:
Ok, good point, I will move it to a variable 👍